components/applications-service/cmd/applications-load-gen/commands/describe.go

```diff
@@ -41,6 +41,8 @@ func runDescribeCmd(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
+	fmt.Print(supGroups.RollupStats())
```

Comment:
This prints something like this at the top of the description. I found it helpful as a sanity check of my load testing configurations.

```
Totals
======
Supervisor groups:  2
Total Supervisors:  6000
Total Services:     18000
HealthCheck rate/s: 600.00
```